Snowflake.Data/Core/SFStatement.cs

```diff
@@ -17,6 +17,61 @@
 
 namespace Snowflake.Data.Core
 {
+    /// <summary>
+    /// The status types of the query.
+    /// </summary>
+    public enum QueryStatus
+    {
+        RUNNING,
+        ABORTING,
+        SUCCESS,
+        FAILED_WITH_ERROR,
+        ABORTED,
+        QUEUED,
+        FAILED_WITH_INCIDENT,
+        DISCONNECTED,
+        RESUMING_WAREHOUSE,
+        // purposeful typo.Is present in QueryDTO.java

```

Comment:
.java?

Reply:
It's a typo from other connectors:
- Java: https://github.com/snowflakedb/snowflake-jdbc/blob/066e25a8718a26b39983b99e0731f3999b906ca6/src/main/java/net/snowflake/client/core/QueryStatus.java#L14
- Python: https://github.com/snowflakedb/snowflake-connector-python/blob/23eda6e8e52e7f0b07997078b99e866f5e29be7f/src/snowflake/connector/constants.py#L345

I'm not sure why but I think the server sends back a misspelled query status

Reply:
The typo is `REPARING` from `QUEUED_REPARING_WAREHOUSE`

The part about QueryDTO.java is now removed